### DISCUSSION TITLE
Error page doesn't work on production (Nginx)

### DISCUSSION THREAD

**Discussion by ChenBri** [nuxt/nuxt#16965] [Questions]
Hello,
I created a custom error.vue page in the source repository just like the docs says ([link](https://v3.nuxtjs.org/guide/features/error-handling/#rendering-an-error-page)). The error page works without any issue locally, but it doesn't work on production using Nginx.
If there's a solution here, it would be great because I couldn't find any successful way to solve it.

After giving up on the error.vue file, I tried creating an error page inside `/pages/error/index.vue`. I then redirected my errors using Nginx configuration: ` error_page 404 /error/index.html;`.
At first, it looked like it worked, but then I found out it created this nuxt's component at the bottom of my screen:
![image](https://user-images.githubusercontent.com/5796466/187925499-418e125c-7ee0-4572-94d9-8e8408104b9c.png)
![image](https://user-images.githubusercontent.com/5796466/187925549-1b84d378-9923-4b61-ad03-e391bded316d.png)
I tried finding a way to remove it with no luck.. I can't find any information about this kind of issue online.

Help would be appreciated.

**Comment by TheAlexLichter** [comment]
Do you use SSG or dynamic SSR?

**Reply by ChenBri** [reply]
Problem solved by adding `generate: {fallback: 'error/index.html'},` to the config and placing `error.vue` inside pages without a subfolder. Also by adding a basic layout according to this post: https://stackoverflow.com/questions/60284864/nuxt-js-static-site-and-404-page
Thanks :)
